app/decorators/address_decorator.rb

```diff
@@ -0,0 +1,15 @@
+class AddressDecorator < Draper::Decorator
+  delegate_all
+
+  def full_name
+    first_name + ' ' + last_name
+  end
+
+  def city_zip
+    city + ' ' + zip
+  end
+
+  def phone
```

Comment:
@b-artem this method is redundant because it doesn't provide any additional logic, you just delegate it to `object`. You can remove it, `delegate_all` does the same